source/dem/dem.cc

```diff
@@ -77,6 +77,7 @@ DEMSolver<dim>::DEMSolver(DEMSolverParameters<dim> dem_parameters)
   , insertion_frequency(parameters.insertion_info.insertion_frequency)
   , standard_deviation_multiplier(2.5)
   , background_dh(triangulation)
+  , n_solids(parameters.solid_objects->number_solids)
```

Comment:
Do we really need this parameter as a class member? We could just use it to resize the vector and instead use the size of the vector everywhere. It would be so much clearer :)

Reply:
Applied.